pages/board.vue

```diff
@@ -1,41 +1,38 @@
+    <div class="max-w-7xl mx-auto spx-6 px-6 lg:px-8 text-lg">
         <h1 class="text-2xl font-bold py-4">Dev Edmonton Society Board</h1>
         <ol>
             <li>
                 <h2 class="text-xl font-bold py-2 border-b border-lightgrey">
                     Board Compostion
                 </h2>
-                <p class="py-2">
-                    The Dev Edmonton Board consists of 7 members filling three
-                    year terms. Every year at the Annual General Meeting (AGM)
-                    elections will be held for any open positions. Details on
-                    how to apply and other information regarding the open roles
-                    will be posted on our
-                    <NuxtLink to="/board_recruiting" class="text-brand-primary"
-                        >board recruitment page</NuxtLink
-                    >
-                    prior to the AGM.
-                </p>
-                <p>
-                    Dev Edmonton Society desires to build a board with diverse
-                    perspectives that reflect our community.
-                </p>
-                <div class="py-2">
-                    <span><strong>Roles</strong></span>
-                    <ul class="list-disc">
-                        <li>President</li>
-                        <li>Secretary</li>
-                        <li>Treasurer</li>
-                        <li>Member at large</li>
-                    </ul>
+                <div class="text-gray-500">
+                    <p class="py-2">
+                        The Dev Edmonton Board consists of 7 members filling
+                        three year terms. Every year at the Annual General
+                        Meeting (AGM) elections will be held for any open
+                        positions. Details on how to apply and other information
+                        regarding the open roles will be posted on our
+                        <NuxtLink
+                            to="/board_recruiting"
+                            class="text-brand-primary"
+                            >board recruitment page</NuxtLink
+                        >
+                        prior to the AGM.
+                    </p>
+                    <p>
+                        Dev Edmonton Society desires to build a board with
+                        diverse perspectives that reflect our community.
+                    </p>
+                    <p>
+                        A new board was elected at the society’s AGM on January
+                        23<sup>rd</sup>
+                        2021. The following members were elected to the board.
+                    </p>
                 </div>
             </li>
-            <li>
-                <h2 class="text-xl font-bold py-2 border-b border-lightgrey">
-                    Current Board
-                </h2>
-                <p class="py-2">
+
+            <!-- <p class="py-2">
```

Comment:
Can you remove this code instead of commenting it out please. Thanks!

Reply:
Yup! sure can.